packages/init/src/node-async-local-storage-patch.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable no-underscore-dangle */
+
+import { AsyncLocalStorage, executionAsyncResource } from 'node:async_hooks';
+
+const { is: ObjectIs } = Object;
+const { apply: ReflectApply } = Reflect;
```

Comment:
Precedent for `objectIs` and `reflectApply` in `ses`’s `commons.js`.

Reply:
Ack, but I wanted to keep the Node style for these intrinsics. Please see the original implementation this is monkey patching.